test/e2e/datepicker.test.js

```diff
@@ -1,14 +1,168 @@
 
-const { assert, driver } = require('vl-ui-core').Test;
+const { assert, driver } = require('vl-ui-core').Test.Setup;
+const { DateTime } = require('luxon');
+const { By } = require('selenium-webdriver');
 const VlDatepickerPage = require('./pages/vl-datepicker.page');
 
 describe('vl-datepicker', async () => {
     const vlDatepickerPage = new VlDatepickerPage(driver);
 
-    before(() => {
+    const now = DateTime.local();
+    const currentDay = now.toFormat('d');
+    const nextDay = now.plus({ day: 1 }).toFormat('d');
+    const currentMonth = now.setLocale('nl').toFormat('LLLL');
+    const currentYear = now.toFormat('yyyy');
+    const currentHour = now.toFormat('H');
+    const currentMinutes = now.toFormat('mm');
+
+    before(async () => {
         return vlDatepickerPage.load();
     });
 
-    after(() => driver && driver.quit());
-  
+    it('als ik enkel een dag selecteer, word automatisch de huidige maand en het huidige jaar gekozen', async () => {
+        const today = now.toFormat('dd.LL.yyyy');
+        const datepicker = await vlDatepickerPage.getDefaultDatepicker();
+        await datepicker.selectDay(currentDay);
+        await assert.eventually.equal(datepicker.getInputValue(), today);
+    });
+
+    it('ik kan een maand selecteren', async () => {
+        const today = now.toFormat('dd.LL.yyyy');
+        const datepicker = await vlDatepickerPage.getDefaultDatepicker();
+        await datepicker.selectMonth(currentMonth);
+        await datepicker.selectDay(currentDay);
+        await assert.eventually.equal(datepicker.getInputValue(), today);
```

Comment:
zou ge (in alle testen) niet beter een andere dag/maand/jaar dan "vandaag" selecteren want stel dat de component default de dag van vandaag selecteert, dan weet je nog niet zeker of het selecteren van een dag/maand/jaar gelukt is